sass/tji/_general.scss

```diff
@@ -715,6 +715,7 @@ hr.story-divider {
 			width: 25%;
 			box-shadow: -2px 0 3px rgba(65,65,65,0.5);
 			min-height: calc(100vh - 100px);
+			min-width: 300px;
```

Comment:
This forces the filter section to run off the page between the 992 and 1024 breakpoints. 

![screen shot 2018-07-01 at 10 33 25 pm](https://user-images.githubusercontent.com/3039125/42144084-cdc65030-7d7e-11e8-9ee4-87d815934d0c.png)

I think using flexbox for the layout might help with this -- but for now, could you allow the filter to wrap or explore another solution?